stats/system_tag.go

```diff
@@ -35,9 +36,54 @@ type SystemTagSet uint32
 // which system tags and non-system tags should be included with with metrics.
```

Comment:
Unrelated but let's fix it. Maybe:
```suggestion
// of which system tags and non-system tags to include.
```
?